client/src/components/ManageArtist/ArtistFormLocation.tsx

```diff
@@ -54,14 +54,17 @@ const ArtistFormLocation: React.FC<{ isManage: boolean }> = ({ isManage }) => {
     return (
       <div
         className={css`
-          margin-bottom: 0.5rem;
+          margin-bottom: 0rem;
```

Comment:
This is a default thing no? Does it need to be explicitly defined?

Reply:
I think it does, I'll verify tomorrow I might be wrong.